hack/local-up-kubeedge.sh

```diff
@@ -42,9 +42,10 @@ function uninstall_kubeedge {
   [[ -n "${CLOUDCORE_PID-}" ]] && sudo kill "${CLOUDCORE_PID}" 2>/dev/null
 
   # kill the edgecore
-  if [[ -n "${EDGECORE_PID-}" ]]; then
-      sudo kill "${EDGECORE_PID}" 2>/dev/null
-  fi
+  [[ -n "${EDGECORE_PID-}" ]] && sudo kill "${EDGECORE_PID}" 2>/dev/null
+
+  # delete data
+  rm -rf /etc/kubeedge /var/lib/kubeedge
```

Comment:
TODO: We shall probably avoid using /etc/kubeedge and /var/lib/kubeedge for testing. As it may result in breaking user's stable config.

Reply:
I will change it.

Reply:
done.